eodhp-web-presence/templates/migrate-job.yaml

```diff
@@ -7,29 +7,21 @@ spec:
     spec:
       containers:
       - name: db-migrate
-        image: {{ .Values.image | default "public.ecr.aws/n1b3o1k2/eodhp-web-presence:0.1.12" }}
+        image: {{ .Values.web.image }}
         imagePullPolicy: Always
         command: ["sh", "-c", "python manage.py migrate"]
+        envFrom:
+            {{- toYaml .Values.web.extraEnvFrom | nindent 12 }}
         env:
-          - name: SQL_HOST
-            value: {{ .Values.database.sql_host | default "postgres-service" }}
-          - name: SQL_PORT
-            value: {{ .Values.database.sql_port | default "5432" | quote }}
-          - name: SQL_DATABASE
-            value: {{ .Values.database.sql_database | default "web_presence_db" }}
-          - name: SQL_ENGINE
-            value: {{ .Values.database.sql_engine | default "django.db.backends.postgresql_psycopg2" }}
-          - name: SQL_USER
-            value: {{ .Values.database.sql_user | default "admin" }}
-          {{- if and (.Values.database.secretKeys.sqlPasswordKey) (.Values.database.existingSecret) }}
+          {{- if and (.Values.web.database.secretKeys.sqlPasswordKey) (.Values.web.database.existingSecret) }}
           - name: SQL_PASSWORD
             valueFrom:
               secretKeyRef:
-                name: {{ .Values.database.existingSecret }}
-                key:  {{ .Values.database.secretKeys.sqlPasswordKey }}
+                name: {{ .Values.web.database.existingSecret }}
+                key:  {{ .Values.web.database.secretKeys.sqlPasswordKey }}
           {{- else }}
```

Comment:
I don't think you need this else statement. If user wants to include password as plain text in values file they can add it to env vars. It would be a user error for user to include it there and also specify existing secret and sql password key which would be reported by kubernetes as a duplicate env var in chart.

Reply:
Removed